File: Java/app/src/main/java/dev/ogabek/java/activity/MainActivity.java
package dev.ogabek.java.activity;

import androidx.appcompat.app.AppCompatActivity;
import androidx.recyclerview.widget.GridLayoutManager;
import androidx.recyclerview.widget.RecyclerView;

import android.os.Bundle;

import java.util.ArrayList;

import dev.ogabek.java.R;
import dev.ogabek.java.adapter.ChatAdapter;
import dev.ogabek.java.model.Chat;

public class MainActivity extends AppCompatActivity {

    private RecyclerView recyclerView;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        initViews();

    }

    private void initViews() {
        recyclerView = findViewById(R.id.rv_main);
        recyclerView.setLayoutManager(new GridLayoutManager(this, 1));

        refreshAdapter(getAllChats());
    }

    private void refreshAdapter(ArrayList<Chat> chats) {
        ChatAdapter adapter = new ChatAdapter(chats);
        recyclerView.setAdapter(adapter);
    }

    private ArrayList<Chat> getAllChats() {
        ArrayList<Chat> chats = new ArrayList<>();
        chats.add(new Chat(R.drawable.user, "Ogabek Matyakubov", true));
        chats.add(new Chat(R.drawable.user2, "OgabekDev", true));
        chats.add(new Chat(R.drawable.user3, "Ogabek", false));
        chats.add(new Chat(R.drawable.user4, "Android Bootcamp B13", false));
        chats.add(new Chat(R.drawable.user, "Ogabek Matyakubov", false));
        chats.add(new Chat(R.drawable.user2, "OgabekDev", true));
        chats.add(new Chat(R.drawable.user3, "Ogabek", false));
        chats.add(new Chat(R.drawable.user4, "Android Bootcamp B13", false));
        chats.add(new Chat(R.drawable.user, "Ogabek Matyakubov", true));
        chats.add(new Chat(R.drawable.user2, "OgabekDev", false));
        chats.add(new Chat(R.drawable.user3, "Ogabek", true));
        chats.add(new Chat(R.drawable.user4, "Android Bootcamp B13", false));
        chats.add(new Chat(R.drawable.user, "Ogabek Matyakubov", true));
        chats.add(new Chat(R.drawable.user2, "OgabekDev", false));
        chats.add(new Chat(R.drawable.user3, "Ogabek", false));
        chats.add(new Chat(R.drawable.user4, "Android Bootcamp B13", false));
        return chats;
    }
}

File: Kotlin/app/src/main/java/dev/ogabek/kotlin/activity/MainActivity.kt
package dev.ogabek.kotlin.activity

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle
import androidx.recyclerview.widget.GridLayoutManager
import androidx.recyclerview.widget.RecyclerView
import dev.ogabek.kotlin.R
import dev.ogabek.kotlin.adapter.ChatAdapter
import dev.ogabek.kotlin.model.Chat

class MainActivity : AppCompatActivity() {
    private lateinit var recyclerView: RecyclerView

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        initViews()

    }

    private fun initViews() {
        recyclerView = findViewById(R.id.rv_main)
        recyclerView.layoutManager = GridLayoutManager(this, 1)

        refreshAdapter(getAllChats())
    }

    private fun refreshAdapter(chats: java.util.ArrayList<Chat>) {
        val adapter = ChatAdapter(chats)
        recyclerView.adapter = adapter
    }

    private fun getAllChats(): ArrayList<Chat> {
        val chats: ArrayList<Chat> = ArrayList()
        chats.add(Chat(R.drawable.user, "Ogabek Matyakubov", true))
        chats.add(Chat(R.drawable.user2, "OgabekDev", true))
        chats.add(Chat(R.drawable.user3, "Ogabek", false))
        chats.add(Chat(R.drawable.user4, "Android Bootcamp B13", false))
        chats.add(Chat(R.drawable.user, "Ogabek Matyakubov", false))
        chats.add(Chat(R.drawable.user2, "OgabekDev", true))
        chats.add(Chat(R.drawable.user3, "Ogabek", false))
        chats.add(Chat(R.drawable.user4, "Android Bootcamp B13", false))
        chats.add(Chat(R.drawable.user, "Ogabek Matyakubov", true))
        chats.add(Chat(R.drawable.user2, "OgabekDev", false))
        chats.add(Chat(R.drawable.user3, "Ogabek", true))
        chats.add(Chat(R.drawable.user4, "Android Bootcamp B13", false))
        chats.add(Chat(R.drawable.user, "Ogabek Matyakubov", true))
        chats.add(Chat(R.drawable.user2, "OgabekDev", false))
        chats.add(Chat(R.drawable.user3, "Ogabek", false))
        chats.add(Chat(R.drawable.user4, "Android Bootcamp B13", false))
        return chats
    }
}